Library/Homebrew/dev-cmd/bump-revision.rb

```diff
@@ -1,76 +1,75 @@
-# typed: true
+# typed: strict
 # frozen_string_literal: true
 
+require "abstract_command"
 require "formula"
 require "cli/parser"
 
 module Homebrew
-  module_function
+  module DevCmd
+    class BumpRevision < AbstractCommand
```

Comment:
Similar to above: I think it would make sense to move this either to `Homebrew::CLI::BumpRevisionCommand` (possibly with `self.class.dev_command?`), or to `Homebrew::CLI::DevCmd::BumpRevision`.

Reply:
> `Homebrew::CLI::BumpRevisionCommand` (possibly with `self.class.dev_command?`), or to `Homebrew::CLI::DevCmd::BumpRevision`.

👎🏻 from me, I prefer the `DevCmd` namespacing as it:
- better mirrors what's on disk
- has fewer nested modules than the second option

Reply:
> better mirrors what's on disk

Renaming the files to follow the module naming would naturally be the consequence.